readme.md

```diff
@@ -187,6 +187,24 @@ normalizeUrl('www.sindresorhus.com?foo=bar&ref=test_ref', {
 //=> 'http://sindresorhus.com/?foo=bar'
 ```
 
+If a boolean is provided, `true` will remove all the query parameters.
+
+```js
+normalizeUrl('www.sindresorhus.com?foo=bar', {
+	removeQueryParameters: true
+});
+//=> 'http://sindresorhus.com'
+```
+
+`false` will not remove any query parameter. Other options like `sortQueryParameters` are still applied when toggled.
```

Comment:
I'm not sure I see the point of:

> Other options like `sortQueryParameters` are still applied when toggled.

That should already be obvious.

Reply:
Removed it.